conda/recipes/cuspatial/meta.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set minor_version =  version.split('.')[0] + '.' + version.split('.')[1] %}
+{% set git_revision_count=environ.get('GIT_DESCRIBE_NUMBER', 0) %}
+{% set py_version=environ.get('CONDA_PY', 36) %}
+
+package:
+  name: cuspatial
+  version: {{ version }}
+
+source:
+  path: ../../..
+
+build:
+  number: {{ git_revision_count }}
+  string: py{{ py_version }}_{{ git_revision_count }}
+
+requirements:
+  build:
+    - python
+    - cython >=0.29,<0.30
+    - setuptools
+    - numba >=0.45.1
+    - dlpack
+    - pyarrow 0.14.1.*
+    - libcudf {{ version }}
```

Comment:
No dependency on `cudf`?

Reply:
Since we are building cudf/libcudf from source as part of the conda build process, I'm assuming it should not be part of the dependency list.

Reply:
It will eventually, but not quite yet.